R/slide.R

```diff
@@ -88,6 +88,25 @@
 #' @importFrom lubridate days weeks
 #' @importFrom rlang .data .env !! enquo enquos sym
 #' @export
+#' @examples 
+#'  # slide a 7-day trailing average formula on cases
+#'   jhu_csse_daily %>%
+#'   group_by(geo_value) %>%
+#'   epi_slide(cases_7dav = mean(cases), n = 7, 
+#'             align = "right")
+#'  
+#'  # slide a left-aligned 7-day trailing average
+#'   jhu_csse_daily %>%
+#'   group_by(geo_value) %>%
+#'   epi_slide(cases_7dav = mean(cases), n = 7, 
+#'             align = "left")
```

Comment:
@ChloeYou

Reply:
Good catch! The word "trailing" is removed from the left-aligned example.

I wasn't able to recreate the print issue. The new column `cases_7dav` was able to print for me:
```
> a = jhu_csse_daily_subset %>%
+    group_by(geo_value) %>%
+    epi_slide(cases_7dav = mean(cases), n = 7, 
+              align = "left")
> print(a)
An `epi_df` object, with metadata:
* geo_type  = state
* time_type = day
* as_of     = 2022-05-20 11:39:30

# A tibble: 60 x 7
# Groups:   geo_value [4]
   geo_value time_value case_rate_7d_av death_rate_7d_av cases cases_7d_av cases_7dav
 * <chr>     <date>               <dbl>            <dbl> <dbl>       <dbl>      <dbl>
 1 ca        2020-06-01            6.85            0.161  2439       2695.      2581.
 2 ca        2020-06-02            6.83            0.184  2421       2688.      2555 
 3 ca        2020-06-03            6.67            0.172  2221       2624.      2611.
 4 ca        2020-06-04            6.98            0.168  3030       2749.      2736.
 5 ca        2020-06-05            6.98            0.166  3130       2747       2734.
 6 ca        2020-06-06            6.67            0.169  2807       2626.      2830 
 7 ca        2020-06-07            6.74            0.165  2022       2655.      2909.
 8 ca        2020-06-08            6.68            0.159  2254       2628.      2946 
 9 ca        2020-06-09            6.82            0.161  2812       2684.      2955.
10 ca        2020-06-10            7.14            0.175  3100       2810.      2979.
# ... with 50 more rows
```

Perhaps I'm missing something? Can you maybe elaborate a bit more? Thank you!

Reply:
Seems like it's system or version dependent.  Maybe because I have
```
> getOption("width")
[1] 80
```
?  Or maybe it's a version thing (I am on `tibble_3.1.7`, R version 4.0.5 (2021-03-31)).

You can probably recreate what I'm seeing with `withr::with_options(list(width=80L), print(a))`.

<details>
  <summary>Experiments:</summary>

```
> 
> print(a)
An `epi_df` object, with metadata:
* geo_type  = state
* time_type = day
* as_of     = 2022-05-20 11:39:30

# A tibble: 60 × 7
# Groups:   geo_value [4]
   geo_value time_value case_rate_7d_av death_rate_7d_av cases cases_7d_av
 * <chr>     <date>               <dbl>            <dbl> <dbl>       <dbl>
 1 ca        2020-06-01            6.85            0.161  2439       2695.
 2 ca        2020-06-02            6.83            0.184  2421       2688.
 3 ca        2020-06-03            6.67            0.172  2221       2624.
 4 ca        2020-06-04            6.98            0.168  3030       2749.
 5 ca        2020-06-05            6.98            0.166  3130       2747 
 6 ca        2020-06-06            6.67            0.169  2807       2626.
 7 ca        2020-06-07            6.74            0.165  2022       2655.
 8 ca        2020-06-08            6.68            0.159  2254       2628.
 9 ca        2020-06-09            6.82            0.161  2812       2684.
10 ca        2020-06-10            7.14            0.175  3100       2810.
# … with 50 more rows, and 1 more variable: cases_7dav <dbl>
> print(a, width=200L)
An `epi_df` object, with metadata:
* geo_type  = state
* time_type = day
* as_of     = 2022-05-20 11:39:30

# A tibble: 60 × 7
# Groups:   geo_value [4]
   geo_value time_value case_rate_7d_av death_rate_7d_av cases cases_7d_av
 * <chr>     <date>               <dbl>            <dbl> <dbl>       <dbl>
 1 ca        2020-06-01            6.85            0.161  2439       2695.
 2 ca        2020-06-02            6.83            0.184  2421       2688.
 3 ca        2020-06-03            6.67            0.172  2221       2624.
 4 ca        2020-06-04            6.98            0.168  3030       2749.
 5 ca        2020-06-05            6.98            0.166  3130       2747 
 6 ca        2020-06-06            6.67            0.169  2807       2626.
 7 ca        2020-06-07            6.74            0.165  2022       2655.
 8 ca        2020-06-08            6.68            0.159  2254       2628.
 9 ca        2020-06-09            6.82            0.161  2812       2684.
10 ca        2020-06-10            7.14            0.175  3100       2810.
   cases_7dav
 *      <dbl>
 1      2581.
 2      2555 
 3      2611.
 4      2736.
 5      2734.
 6      2830 
 7      2909.
 8      2946 
 9      2955.
10      2979.
# … with 50 more rows
> withr::with_options(list(width=200L), print(a))
An `epi_df` object, with metadata:
* geo_type  = state
* time_type = day
* as_of     = 2022-05-20 11:39:30

# A tibble: 60 × 7
# Groups:   geo_value [4]
   geo_value time_value case_rate_7d_av death_rate_7d_av cases cases_7d_av cases_7dav
 * <chr>     <date>               <dbl>            <dbl> <dbl>       <dbl>      <dbl>
 1 ca        2020-06-01            6.85            0.161  2439       2695.      2581.
 2 ca        2020-06-02            6.83            0.184  2421       2688.      2555 
 3 ca        2020-06-03            6.67            0.172  2221       2624.      2611.
 4 ca        2020-06-04            6.98            0.168  3030       2749.      2736.
 5 ca        2020-06-05            6.98            0.166  3130       2747       2734.
 6 ca        2020-06-06            6.67            0.169  2807       2626.      2830 
 7 ca        2020-06-07            6.74            0.165  2022       2655.      2909.
 8 ca        2020-06-08            6.68            0.159  2254       2628.      2946 
 9 ca        2020-06-09            6.82            0.161  2812       2684.      2955.
10 ca        2020-06-10            7.14            0.175  3100       2810.      2979.
# … with 50 more rows
```
</details>

Reply:
See #83